### PR TITLE
chore (#7373)  UI : Remove momentjs and use luxon

### LINKED ISSUE #7373: UI : Remove momentjs and use luxon
Remove momentjs and use luxon

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Closes #7373
### Describe your changes :

I worked on issue #7373 and also removed the `react-google-login` as we are no longer using it.

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

**Review by chirag-madlani — COMMENTED** [review]
@Sachin-chaurasiya It's like we are manipulating dateTime in all the files can we separate out them to a util for `DateTimeUtil.ts`?

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Bots.spec.js [hunk 1]
```diff
@@ -32,10 +31,7 @@ const expirationTime = {
 };
 
 const createExpiryDate = (expiry, days) => {
-  const currentDate = Date.now();
-  const endDate = moment(currentDate, 'x')
-    .add(expiry, days)
-    .format(`ddd Do MMMM, YYYY,hh:mm A.`);
+  const endDate = getExpiryDateTimeFromDate(expiry, days);
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Bots.spec.js:34]: We don't need this function anymore `createExpiryDate`

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya It's like we are manipulating dateTime in all the files can we separate out them to a util for `DateTimeUtil.ts`?

@chirag-madlani , Done.

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Bots.spec.js (hunk 1 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/Bots.spec.js]: Done.

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**